src/conn/conn_prefetch.c

```diff
@@ -120,17 +117,14 @@ __wt_prefetch_thread_run(WT_SESSION_IMPL *session, WT_THREAD *thread)
          * It probably isn't strictly necessary to re-acquire the lock to reset the flag, but other
```

Comment:
This comment needs updating, something like:
"We don't take the prefetech_lock here as it protects the queue, not the busy flag."